### PR TITLE
[IT-1186] Add shared Nextflow production account under ScienceOU

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
I'm opening this PR to create a production AWS account for running Nextflow workflows. The intent is that the production account will be used on PHI data, so it needs to be HIPAA compliant. I expect there to be manual steps once this PR is merged. 

See #131 for the PR creating the corresponding development account. 

----

PR Checklist:
- [x] Describe and explain your intentions for this change
- [x] Setup pre-commit and run the validators (info in README.md)
  To validate files run: `pre-commit run --all-files`

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -260,3 +262,27 @@ Organization:
         Project: Infrastructure
         budget-alarm-threshold: 20000
         budget-alarm-threshold-email-recipient: aws-mobilehealth-dataengineering@sagebase.org
+
+  NextflowWorkflowsSharedDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: nextflow-workflows-shared-dev
+      RootEmail: aws-nextflow-workflows@sagebase.org
```
Comment (zaro0508) [inline org-formation/organization.yaml:270]: AWS requires a unique email address for every account.

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -260,3 +262,27 @@ Organization:
...
+      Alias: org-sagebase-nextflow-workflows-shared-dev
```
Comment (zaro0508) [inline org-formation/organization.yaml:271]: do we really need `shared`?  seems kinda implicit to me.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Comment by zaro0508** [comment]
would you mind splitting this to two PRs? one for each account?

**Comment by BrunoGrandePhD** [comment]
@zaro0508: I've restricted this PR to the production account. See #131 for the development account. I've made sure different root emails are used, and I've removed the implied `shared` from the names. Thanks!

**Review by zaro0508 — APPROVED** [review]

**Review by ahayden — APPROVED** [review]
There will be some manual steps to take care of after this.

File: org-formation/organization.yaml [hunk 3]
```diff
@@ -35,6 +35,7 @@ Organization:
       Accounts:
         - !Ref ScipoolProdAccount
         - !Ref ScicompAccount
+        - !Ref NextflowWorkflowsProdAccount
```
[the remark this answers is not in the document]
Comment (BrunoGrandePhD) [reply org-formation/organization.yaml]: I prefer your naming convention. And I expect this account to be reserved for Nextflow-related resources. 

@tthyer: Let us know if you have something else in mind.

**Comment by BrunoGrandePhD** [comment]
I'm going with the CompOnc imCORE project for now since this work will be used to deploy workflows for processing imCORE data. This might change in a separate PR if I'm instructed of a better department/project for an AWS account shared between SysBio and CompOnc.

**Pull request merged**